assets/js/components/Collection/Form.jsx

```diff
@@ -111,104 +114,71 @@ const CollectionForm = ({ collection }) => {
           </div>
         </div>
 
-        <div className="field">
-          <label htmlFor="collection-type" className="label">
-            Collection Type
-          </label>
-          <div className="control">
-            <div className="select">
-              <select
-                ref={register}
-                id="collection-type"
-                name="collectionType"
-                data-testid="input-collection-type"
-              >
-                <option>NUL Collection</option>
-                <option>NUL Theme</option>
-              </select>
-            </div>
-          </div>
-          <p className="help">Type of Collection</p>
-        </div>
-
-        <div className="field">
-          <p className="notification is-warning">
-            TODO: Wire up [Select thumbnail]
-          </p>
-        </div>
+        <UIFormFieldAndDisplay label="Collection Type">
+          <UIFormSelect
+            register={register}
+            name="collectionType"
+            label="Collection Type"
+            options={COLLECTION_TYPES}
+            defaultValue={collection ? collection.collectionType : ""}
+            errors={errors}
+            data-testid="input-collection-type"
+          />
+          <UITagNotYetSupported label="Display not yet supported" />
+          <UITagNotYetSupported label="Update not yet supported" />{" "}
+        </UIFormFieldAndDisplay>
 
-        <div className="field">
-          <label htmlFor="description" className="label">
-            Description
-          </label>
-          <div className="control">
-            <textarea
-              ref={register}
-              name="description"
-              id="description"
-              defaultValue={collection ? collection.description : ""}
-              className="textarea"
-              rows="8"
-              data-testid="textarea-description"
-            ></textarea>
-          </div>
-          <p className="help">Describe the Collection</p>
-        </div>
+        <UIFormFieldAndDisplay label="Description">
+          <UIFormTextarea
+            register={register}
+            errors={errors}
+            name="description"
+            label="Description"
+            defaultValue={collection ? collection.description : ""}
+            className="textarea"
```

Comment:
We don't need to pass in these class names.  The components already have them.   Can we remove in all occurences?

Reply:
ok!